src/config.rs

```diff
@@ -24,7 +24,9 @@ const CONFIG_FILE_NAME: &str = "spotifyd.conf";
     feature = "pulseaudio_backend",
     feature = "portaudio_backend",
     feature = "alsa_backend",
-    feature = "rodio_backend"
+    feature = "rodio_backend",
+    feature = "rodiojack_backend",
+    feature = "jackaudio_backend",
```

Comment:
I think, this might be a leftover from your testing? As this feature does not actually exist in the `Cargo.toml`.